routes/graphqlVariablesExample.js

```diff
@@ -0,0 +1,61 @@
+const { request, gql } = require('graphql-request')
+
```

Comment:
@leomrocha We can probably safely delete this file?

Reply:
@EvgeniiaVak absolutely, we need to delete it

Reply:
Done